addons/mrp_account/wizard/mrp_wip_accounting.py

```diff
@@ -51,7 +50,7 @@ def default_get(self, fields_list):
             if default:
                 res['journal_id'] = default.id
         if 'reference' in fields_list:
-            res['reference'] = _("Manufacturing WIP - %(orders_list)s", orders_list=productions and format_list(self.env, productions.mapped('name')) or _("Manual Entry"))
+            res['reference'] = _("Manufacturing WIP - %(orders_list)s", orders_list=productions and productions.mapped('name') or _("Manual Entry"))
```

Comment:
mapped on empty recordset will generate an empty list, which can be directly used with `or`
```suggestion
            res['reference'] = _("Manufacturing WIP - %(orders_list)s", orders_list=productions.mapped('name') or _("Manual Entry"))
```

Reply:
Right. Updated.